spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfiguration.java

```diff
@@ -81,6 +81,7 @@ public ReactiveDiscoveryClientHealthIndicator kubernetesReactiveDiscoveryClientH
 		return healthIndicator;
 	}
 
+	@Deprecated(forRemoval = true)
 	@Bean
```

Comment:
Should these annotations be removed?

Reply:
you're somehow looking at an outdated file. Here is the latest one in the PR: https://github.com/spring-cloud/spring-cloud-kubernetes/blob/727ed0bbf10c50d67332f30205fc2b072f84d3b6/spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfiguration.java